lib/components/narrative/default/default-itinerary.js

```diff
@@ -88,20 +93,22 @@ function ItineraryDescription({ itinerary }) {
   })
 
   const mainMode = <FormattedMode mode={accessModeId} />
-  return transitMode
-    ? <FormattedMessage
-      id='components.DefaultItinerary.multiModeSummary'
+  return transitMode ? (
+    <FormattedMessage
+      id="components.DefaultItinerary.multiModeSummary"
       values={{ accessMode: mainMode, transitMode }}
     />
-    : mainMode
+  ) : (
+    mainMode
+  )
 }
 
 const ITINERARY_ATTRIBUTES = [
   {
     alias: 'best',
     id: 'duration',
     order: 0,
-    render: (itinerary, options) => (
+    render: (itinerary /*, options */) => (
```

Comment:
Should we remove this rather than comment if we're not using options anymore?

Reply:
Yeah I removed them in https://github.com/opentripplanner/otp-react-redux/pull/626/commits/93ce1d67362de7e7a11478c731c5283bba640837.